source/javascripts/lib/wirsing/filters/switch.coffee

```diff
@@ -0,0 +1,17 @@
+@wirsing ?= {}
+wirsing.filter ?= {}
```

Comment:
Hmm… in Coffeescript even something like this would be possible:

```coffeescript
module 'wirsing.filters',
  switch: # …
```

This is actually what I like best so far. What do you think?